File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/NetworkRequest.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class NetworkRequest(
  val url: String,
  val body: String? = null,
  val error: String? = null,
)

File: agent/bindings/csharp/NetworkRequest.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class NetworkRequest
  {
    [JsonProperty(PropertyName = "url")]
    public string Url { get; set; }
    [JsonProperty(PropertyName = "body")]
    public string Body { get; set; }
    [JsonProperty(PropertyName = "error")]
    public string Error { get; set; }
  }
}
